docs/reference/api/rest/introduction.md

```diff
@@ -46,4 +49,177 @@ Chances are you're looking for how to interact with **Items**, the atomic conten
 </div>
 </div>
 
----
\ No newline at end of file
+---
+
+## Relational Data
+
+<div class="two-up">
+<div class="left">
+
+### Fetching
+
+By default, Directus only retrieves the "root level" of an item. Relational data can be nested by using
+[the `fields` parameter](/reference/api/query/#fields).
+
+You can use a wildcard (`*`) to fetch all items in a given level, and use `*.*` to fetch everything on the next related
```

Comment:
You can use a wildcard (`*`) to fetch all items in a given level, use `*.*` to fetch everything on the next relational level, and so on, to any depth required.